core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -350,11 +350,17 @@ private boolean processInProgress(TransferProcess process) {
             } else {
                 // Process is not finished yet, so it stays in the IN_PROGRESS state
                 monitor.info(format("Transfer process %s not COMPLETED yet. The process will not advance to the COMPLETED state.", process.getId()));
+                breakLease(process);
                 return false;
             }
         }
     }
 
+    private void breakLease(TransferProcess process) {
```

Comment:
nit: to keep consistent order of methods in the class this method should be moved to the bottom of the class after `processConsumerRequest` method because it's called there as well